Read-Only File System on Mac (/usr/bin is write protected)

When following the instructions to install on a Mac (OS Monterey) (https://github.com/openstreetmap/mod_tile/blob/master/docs/build/building_on_macos.md), the following issue occurs (after running command `sudo cmake --install . --prefix /usr --strip`):

```
-- Install configuration: "Release"
-- Up-to-date: /var/cache/renderd/tiles
-- Installing: /var/run/renderd
-- Installing: /opt/local/etc/apache2/extra/httpd-tile.conf
-- Installing: /etc/renderd.conf
-- Installing: /opt/local/lib/apache2/modules/mod_tile.so
-- Installing: /usr/bin/render_expired
CMake Error at cmake_install.cmake:110 (file):
  file INSTALL cannot copy file "/tmp/mod_tile_build/src/render_expired" to
  "/usr/bin/render_expired": Read-only file system.
```

As I understand, Catalina (and other newer ones) has a new file system arrangement where most of the system files are write-protected. Apple uses two partitions, a read-only one for the main system files and a writeable one for other files, and melds them together and presents them as one to the user.

What can be done in this case?

--- a/docs/build/building_on_macos.md
+++ b/docs/build/building_on_macos.md
@@ -43,7 +43,7 @@ cmake -B . -S /tmp/mod_tile_src \
   -DENABLE_TESTS:BOOL=ON
 cmake --build .
 ctest
-sudo cmake --install . --strip
+sudo cmake --install . --prefix /usr/local --strip
 
 # Create /usr/local/share/renderd directory
 sudo mkdir -p /usr/local/share/renderd

--- a/docs/build/building_on_macos.md
+++ b/docs/build/building_on_macos.md
@@ -43,7 +43,7 @@ cmake -B . -S /tmp/mod_tile_src \
   -DENABLE_TESTS:BOOL=ON
 cmake --build .
 ctest
-sudo cmake --install . --prefix /usr --strip
+sudo cmake --install . --strip
 
 # Create /usr/local/share/renderd directory
 sudo mkdir -p /usr/local/share/renderd

